config/cert-manager/manager_certmanager_patch.yaml

```diff
@@ -0,0 +1,15 @@
+# This patch adds an environment variable to the workspace deployment
+# to allow it to figure out which secret to use for the webhooks deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: manager
+  namespace: system
```

Comment:
I'm not sure what the right way to fix it but it seems `namespace: system` here and in other places leads to creation of `devworkspace-controller-system` namespace which then is not used anywhere.
```
 > make install                                                                           Current env vars:
    NAMESPACE=ws-operator
    IMG=sleshchenko/che-workspace-controller:local
namespace/ws-operator created
deployment.apps/che-plugin-registry created
...
namespace/devworkspace-controller-system created
...
```

Reply:
This was due to `manager.yaml` containing an explicit `system` namespace it its yaml. This namespace would get prefixed like everything else and created.